math/src/unsigned_integer/element.rs

```diff
@@ -963,7 +969,8 @@ mod tests_u384 {
 
     proptest! {
         #[test]
-        fn bitand(a in any::<[u64; N_LIMBS]>(), b in any::<[u64; N_LIMBS]>()) {
+        #[cfg(feature = "proptest")]
```

Comment:
Isn't `proptest` always enabled for tests? In any case, you'd probably save time and code by flagging the whole `proptest` block instead.

Reply:
lol yeah that makes a ton of sense thank you!

Reply:
Followed your suggestion for flagging!